modulemd/modulemd-modulemetadata.c

```diff
@@ -121,6 +121,7 @@ modulemd_modulemetadata_set_name (ModulemdModuleMetadata *self,
                                   const gchar            *name)
 {
   g_return_if_fail (MODULEMD_IS_MODULEMETADATA (self));
+  g_return_if_fail (name);
```

Comment:
it's fine, it is in separate patch which you can drop. I was surprised to see default value as `""`, but accepting to set it to `NULL`.